addons/website/views/website_templates.xml

```diff
@@ -487,7 +487,7 @@
                         </t>
                         <!-- Nav -->
                         <t t-call="website.navbar_nav">
-                            <t t-set="_nav_class" t-valuef="mx-auto order-first order-lg-12"/>
+                            <t t-set="_nav_class" t-valuef="mx-auto order-first order-lg-5"/>
```

Comment:
Just wondering: if there is an `order-first` variable, I guess there is an `order-last` ? Should not this one be used instead of `order-5` ?

Reply:
Actually `order-last` is already used by the CTA button, the navbar must be positioned between the `order-first` and the `order-last` elements